### PR TITLE
Update project termination doc

### PR DESCRIPTION
**Pull request by pabrahamsson**
Document more steps to troubleshoot why your project is stuck in status `Terminating`

### CODE REVIEW

**Comment by garethahealy** [comment]
@pabrahamsson ; you OK to just update the description of the PR for future reference?

File: fix-project-terminating/README.md [hunk 1]
```diff
@@ -2,6 +2,47 @@
...
+> oc get project <project-name> -o yaml
+```
+
+Look in the `status` of the output to see what needs to be patched. In the example below we'll need to patach `cephclusters.ceph.rook.io`
+
+```
+...
+status:
+  conditions:
+  - lastTransitionTime: "2020-03-26T10:01:05Z"
```
Comment (oybed) [inline fix-project-terminating/README.md]: Do we feel this section is all needed, or do we rather just cut it down to the one that shows the message of interest? I.e.: the "Some content in the namespace ..."

Comment (pabrahamsson) [reply fix-project-terminating/README.md]: Sure, updated

File: fix-project-terminating/README.md [hunk 2]
```diff
@@ -2,6 +2,26 @@
 
 In some cases, projects/namespaces get stuck in a `terminating` state. In most cases this is due to one or more finalizers on objects within the projects  - including for Custom Resources (CRs) based on Custom Resource Definitions (CRDs). For these, the following `patch` command can be used to remove the finalizer and hence recover from the terminating state.
 
+### Find out what's holding up the termination of the project
+
+```bash
+> oc get project <project-name> -o yaml
+```
+
+Look in the `status` of the output to see what needs to be patched. In the example below we'll need to patach `cephclusters.ceph.rook.io`
```
Comment (oybed) [inline fix-project-terminating/README.md]: typo: `patach`

Comment (pabrahamsson) [reply fix-project-terminating/README.md]: Thanks, updated

**Pull request merged**